composer.json

```diff
@@ -67,5 +68,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
```

Comment:
Seems a bit overkill but yeah might be better, we need to find and validate a neat solution before I can implement it. ping @api-platform/core-team